src/posts/faq.md

```diff
@@ -18,15 +18,12 @@ AI safety researchers (who are the experts on this topic) are divided on this qu
 Note that no (surveyed) AI safety researchers believe that there's a 0% chance.
 However, there might be selection bias here: people who work in the AI safety field are likely to do so because they believe preventing bad AI outcomes is important.
 
-If you ask AI researchers in general (not safety specialists), this number drops to a [mean value of around 14%](https://aiimpacts.org/2022-expert-survey-on-progress-in-ai/).
+If you ask AI researchers in general (not safety specialists), this number drops to a [mean value of around 14%](https://aiimpacts.org/2022-expert-survey-on-progress-in-ai/), with a median of 5%.
 A small minority, about 4% of them, believe that the alignment problem is not a real problem.
 Note that there might be a selection bias here in the opposite direction: people who work in AI are likely to do so because they believe AI will be beneficial.
 
 _Imagine you're invited to take a test flight on a new airplane_.
-The airplane safety experts on average think there's a 30% chance it would crash.
-The plane engineers think there's a 14% chance of crashing.
-About 4% of engineers are saying that there's no chance of crashing, it is uncrashable.
-
+The average plane engineers thinks there's a 5% chance of crashing.
```

Comment:
Using the median value while saying "average" is misleading. Also, I don't think we should use the median value here - we should use the mean.

Say we had 10 researchers, 4 of which had a p(doom) of 100% and 6 of them 0%. The median p(doom) would be 0%, the mean 40%.

How worried should you be in this case? 40% or 0%? Really, the relevant number here is the _average_, not the median!

The median is misleading.

Reply:
Okay reverted to 14%